docs/csharp/whats-new/csharp-9.md

```diff
@@ -0,0 +1,231 @@
+---
+title: What's new in C# 9.0 - C# Guide
+description: Get an overview of the new features available in C# 9.0.
+ms.date: 09/04/2020
+---
+# What's new in C# 9.0
+
+C# 9.0 adds the following features and enhancements to the C# language:
+
+- Records
+- Init only setters
+- Top-level statements
+- Pattern matching enhancements
+- Native sized integers
+- Function pointers
+- Suppress emitting localsinit flag
+- Target-typed new expressions
+- static anonymous functions
+- Target-typed conditional expressions
+- Covariant return types
+- Lambda discard parameters
+- Attributes on local functions
+- Module initializers
+- New features for partial methods
```

Comment:
source generators is probably missing?

Reply:
Source Generators aren't a C# language feature. They are a new capability.